Havit.Blazor.Components.Web.Bootstrap/TreeViews/HxTreeView.razor.css

```diff
@@ -0,0 +1,7 @@
+.hx-treeview {
+  height: 100%;
+  border-radius: 3px;
+  border: solid 1px lightgray;
+  padding: 5px;
+  overflow-y: auto;
+}
```

Comment:
@crdo Please review the default visual for new HxTreeView component
* do we want the component to have a default border or should we leave the user to add the border on his own using utility-classes or `div` container?
* ...similar to the other settings, should the be default representation of the HxTreeView?